script/DeployRouter.s.sol

```diff
@@ -36,6 +40,16 @@ abstract contract DeployRouter is DeployBase {
             console2.log('Router Deployed:', deployedAddress);
             Router router = Router(deployedAddress);
             console2.log('Router Owner:', router.owner());
+
+            // Set and check signer
+            router.addSigner(SIGNER);
+            require(router.signers(SIGNER), 'Router signer is invalid');
+
+            // Set and check fee rate
+            if (cfg.feeRate > 0) {
+                router.setFeeRate(cfg.feeRate);
+                require(router.feeRate() == cfg.feeRate, 'Router fee rate is invalid');
```

Comment:
wondering if this `require` should be placed after the `if(cfg.feeRate > 0)` clause to ensure `router.feeRate() == 0`

Reply:
Fixed, thanks.